online-store/src/index.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="icon" href="./img/basket.png">
+  <meta name="description" content="Online Store">
+  <title>Online Store</title>
+</head>
+<body class="body">
+  <header class="header">
+    <div class="container">
+      <a class="logo" href="#"></a>
+      <h1 class="title">Online Store</h1>
+      <div class="shopping-cart" title="Корзина">
+        <span class="quantity">0</span>
+      </div>
+    </div>
+  </header>
+  <main class="main container">
+    <div class="filters">
+      <div class="filters-group filters-by-values">
+        <h4 class="title">Фильтры по значению</h4>
+        <div class="filter filter-by-manufacturer">
+          Производитель:
```

Comment:
Imagine you added some new manufacturer, so you have to change your hardcoded html too and it is related no only to manufacturer. Imagine you keep adding new colors, or cameras count. Or you even decided to sell not only phones... In real life data came from the backed so most of the time you can't hardcode too much, I would suggest to render this html dynamically